awswrangler/s3/_write_deltalake.py

```diff
@@ -54,7 +54,7 @@ def to_deltalake(
     mode: Literal["error", "append", "overwrite", "ignore"] = "append",
     dtype: dict[str, str] | None = None,
     partition_cols: list[str] | None = None,
-    overwrite_schema: bool = False,
```

Comment:
deltalake package introduced a breaking change where `overwrite_schema` boolean argument was deprecated and replaced with a literal argument